dataset_config/1000_genomes/bigquery.json

```diff
@@ -12,8 +12,7 @@
 //   The (optional) name of BigQuery column that holds the sample ID.
 // sample_file_columns:
 //   An (optional) map from file type to BigQuery column containing file paths.
```

Comment:
For AMP PD, instead of the sample overview facet, we have separate facets for "Has WGS sample" and "Has RNA sample". We specify the columns here so that generating sql queries for these facets work properly.